### PR TITLE
 Bump cri-tools to v1.0.0-beta.0

### PR DESCRIPTION
**Pull request by feiskyer**
/assign @Random-Liu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Version matrix:
 
 | Kubernetes Version | cri-tools Version | cri-tools branch |
 |--------------------|-------------------|------------------|
-| master             |                   | master           |
+| master             | v1.0.0-beta.0     | master           |
```
Comment (Random-Liu) [inline README.md]: 1.10.X

**Comment by Random-Liu** [comment]
LGTM with one comment. Feel free to self apply the label and merge after you address the comment.

File: README.md (hunk 1 above)
Comment (feiskyer) [reply README.md]: fixed

**Pull request merged**